src/audio/codec_adapter/codec/cadence.c

```diff
@@ -139,6 +139,48 @@ int cadence_codec_init(struct comp_dev *dev)
 	return ret;
 }
 
+static int update_stream_params(struct comp_dev *dev)
+{
+	int ret = 0;
+	struct comp_data *ca_data = comp_get_drvdata(dev);
+	struct codec_data *codec = comp_get_codec(dev);
+	struct cadence_codec_data *cd = codec->private;
+	struct ca_config *ca_config = &ca_data->ca_config;
+	struct sof_ipc_stream_params *stream = &ca_data->stream_params;
+	/* copy stream params localy to ensure container size is correct */
+	uint32_t sample_width = (stream->frame_fmt == SOF_IPC_FRAME_S16_LE) ? 0x10 : 0x20;
+	uint32_t sample_rate = stream->rate;
+	uint32_t channels = stream->channels;
+
+	comp_err(dev, "update_stream_params() start");
+
```

Comment:
We should go with comp_info or comp_dbg here.

Reply:
Yes definitely.

Reply:
I think this still applies.

Reply:
Fixed, thanks